srcpkgs/mac-fdisk/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'mac-fdisk'
+pkgname=mac-fdisk
+version=0.1
+revision=1
+wrksrc=${pkgname}-${version}.orig
+build_style=gnu-makefile
+short_desc="Format and edit Apple Partition Map drives"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="ftp://ftp.mklinux.apple.com/pub/Other_Tools/"
```

Comment:
ftp homepage ?

Reply:
That's what the homepage is on the debian and gentoo packages. It's a pretty old program and the homepage went down ages ago. You want me to set it to the debian package page?
I should also replace only_for_arches with the new thing.